### PR TITLE
sys-fs/bcachefs-tools: Add 1.11.0

### PR DESCRIPTION
**Pull request by csfore**
- Update 9999 to use git:// since https seems to have been disabled (500 error)
- Update cargo dependencies with pycargoebuild
- Remove tests from 1.11.0 since they are no longer in this version

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: sys-fs/bcachefs-tools/bcachefs-tools-1.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+LLVM_COMPAT=( {16..18} )
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit cargo flag-o-matic llvm-r1 python-any-r1 shell-completion toolchain-funcs unpacker
+
+DESCRIPTION="Tools for bcachefs"
+HOMEPAGE="https://bcachefs.org/"
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="git://evilpiepirate.org/bcachefs-tools.git"
```
Comment (thesamesam) [inline sys-fs/bcachefs-tools/bcachefs-tools-1.11.0.ebuild:108]: This is unsafe. Please ask upstream to restore HTTPS.

**Comment by csfore** [comment]
Reverted the -9999 change for now!

**Comment by eli-schwartz** [comment]
Just in time since it works again. :)

Kent said that somehow git-http-backend was complaining about "safe.directory", who'da thunk ;) and the solution was to migrate from passing `-c safe.directory=*` to git-daemon, to simply specifying that in /etc/gitconfig since apache wasn't offering an obvious way to pass args to git-http-backend.

**Comment by Flowdalic** [comment]
> Reverted the -9999 change for now!

Please also revert in -1.11.0. Thanks

**Pull request closed without merging**